unison-runtime/src/Unison/Runtime/Pattern.hs

```diff
@@ -13,7 +13,7 @@ module Unison.Runtime.Pattern
 where
 
 import Control.Monad.State (State, evalState, modify, runState, state)
-import Data.List (transpose)
+import Data.List (nub, transpose)
```

Comment:
`v` should have Ord right? Is it possible to `nubOrd` instead?
If not, no worries.